### ISSUE TITLE
tsconfig paths alias

### ISSUE THREAD

**Issue by sahitono** [hapijs/lab#1033]
#### Support plan

* *is this issue currently blocking your project?* (no):
* *is this issue affecting a production system?* (no):

#### Context

* *node version*: 16.13.2
* *module version*:  24.5.1
* *environment* (e.g. node, browser, native): node
* *used with* (e.g. hapi application, another framework, standalone, ...): hapi application
* *any other relevant information*: used in typescript project

#### How can we help?

Hi, I am currently using tsconfig paths alias like this
```json
        "paths": {
            "~/*": [
                "./*"
            ]
        },
```
but it looks the built in typescript transformer doesn't support tsconfig paths alias yet, Is there any way to integrate the transformer it with tsconfig-paths or tsc-alias?

thank you in advance

**Comment by devinivy** [comment]
I would personally be open to bringing this functionality into lab, and would be happy to review someone's work on this.

**Comment by devinivy** [comment]
This was resolved by #1041, which was published in 24.7.0.
